hilo.rb

```diff
@@ -0,0 +1,26 @@
+random_number = rand(1..100)
+# print random number for testing purposes
+puts random_number
+puts "Enter a number between 1 and 100. press enter to submit your guess"
+guess = $stdin.readline()
```

Comment:
You can also use [`gets`](https://www.rubyguides.com/2019/10/ruby-chomp-gets/).  I think it still uses `$stdin.readline()` under the hood.